src/EditorFeatures/Test/MetadataAsSource/AbstractMetadataAsSourceTests.cs

```diff
@@ -14,10 +16,10 @@ namespace Microsoft.CodeAnalysis.Editor.UnitTests.MetadataAsSource
     public abstract partial class AbstractMetadataAsSourceTests
     {
         internal static async Task GenerateAndVerifySourceAsync(
-            string metadataSource, string symbolName, string projectLanguage, string expected, bool includeXmlDocComments = false, string languageVersion = null, string metadataLanguageVersion = null)
+            string metadataSource, string symbolName, string projectLanguage, IFormattable expected, bool allowDecompilation = false, bool includeXmlDocComments = false, string languageVersion = null, string metadataLanguageVersion = null, ExportProvider exportProvider = null)
```

Comment:
See comments elsewhere: the IFormattable magic here deserves a number of comments about what is going on. It also means debugging this is toxic as the debugger could totally run the ToString() as a side effect and break the test. Can we just make this `Lazy<string>`?

Reply:
Replaced with `Lazy<string>`